statsmodels/iolib/tests/test_table.py

```diff
@@ -144,6 +144,30 @@ def test_ltx_fmt1(self):
 """ % desired[1:-1]
             actual_centered = '\n%s\n' % tbl.as_latex_tabular()
             self.assertEqual(actual_centered, desired_centered)
+    def test_SimpleTable_special_chars(self):
+		# Simple table with characters: (%, >, |, _, $, &, #)
+		cell0c_data = 22
+		cell1c_data = 1053
+		row0c_data = [cell0c_data, cell1c_data]
+		row1c_data = [23, 6250.4]
+		table1c_data = [ row0c_data, row1c_data ]
+		test1c_stubs = ('> stub1 %', 'stub_2')
+		test1c_header = ('#header 1$', 'header&2 |')
+		tbl_c = SimpleTable(table1c_data, test1c_header, test1c_stubs, ltx_fmt=ltx_fmt1)
+        def test_ltx_special_chars(self):
```

Comment:
wrong indentation, unit tests on TravisCI fail
if this is because of tabs:
please use only spaces, no TABs in the code.
use editor setting where tab key adds 4 spaces